_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
```

Comment:
I'd refrain from using v4...v3 is here for so long noone remembers anything before that, imho. So I think something like ApiaryUI—rethought for speed or something like that would fit better.

Reply:
used `New Interactive Documentation`. Dropped v3, v4 references